.golangci.yml

```diff
@@ -16,6 +16,9 @@ issues:
       linters:
         - dupl
         - lll
+    - path: "pkg/client/(.+)_cache.go"
```

Comment:
The code of the three caches is too duplicate to pass the check, but to be honest I didn't find a proper abstraction that doesn't also make it less readable. But I am open to inputs on this.

Reply:
I created https://github.com/glasskube/glasskube/pull/777 to address this. Let me know what you think :slightly_smiling_face:

Reply:
thanks, LGTM :+1: